lib/shopifyHmac/extractHmacParameter.test.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const { test } = require('tap');
+const extractHmacParameter = require('./extractHmacParameter');
+
+test('Extracts a Hmac parameter from a valid query object', async ({
```

Comment:
same, and for the other instances as well
```suggestion
test('Extracts a HMAC parameter from a valid query object', async ({
```